patches/minecraft/net/minecraft/world/entity/item/ItemEntity.java.patch

```diff
@@ -144,3 +145,82 @@
     }
  
     public float m_32008_(float p_32009_) {
+@@ -425,4 +_,78 @@
+    public float m_213816_() {
+       return 180.0F - this.m_32008_(0.5F) / ((float)Math.PI * 2F) * 360.0F;
+    }
++
++   // FORGE START
++   private final java.util.Set<net.minecraftforge.items.pickup.ItemPickupReason> disabledPickupReasons = com.google.common.collect.Sets.newHashSet();
++   private final java.util.Set<net.minecraftforge.items.pickup.ItemPickupPredicate> pickupPredicates = com.google.common.collect.Sets.newHashSet();
```

Comment:
Now that we are past java 7. We use the diamond operator instead of google's helpers.